src/main/java/org/opentripplanner/graph_builder/module/osm/OpenStreetMapModule.java

```diff
@@ -87,25 +88,29 @@ public class OpenStreetMapModule implements GraphBuilderModule {
    * Whether we should create bike P+R stations from OSM data. (default false)
    */
   public boolean staticBikeParkAndRide;
-  public int maxAreaNodes = 500;
+  public int maxAreaNodes = 150;
   /**
    * Whether ways tagged foot/bicycle=discouraged should be marked as inaccessible
    */
-  public boolean banDiscouragedWalking = false;
-  public boolean banDiscouragedBiking = false;
+  private final boolean banDiscouragedWalking;
+  private final boolean banDiscouragedBiking;
 
   public OpenStreetMapModule(
     Collection<OpenStreetMapProvider> providers,
     Set<String> boardingAreaRefTags,
     Graph graph,
     DataImportIssueStore issueStore,
-    boolean areaVisibility
+    boolean areaVisibility,
+    boolean banDiscouragedWalking,
+    boolean banDiscouragedBiking
   ) {
     this.providers = List.copyOf(providers);
     this.boardingAreaRefTags = boardingAreaRefTags;
     this.graph = graph;
     this.issueStore = issueStore;
     this.areaVisibility = areaVisibility;
+    this.banDiscouragedWalking = banDiscouragedWalking;
+    this.banDiscouragedBiking = banDiscouragedBiking;
```

Comment:
Can't we add the `staticParkAndRide`, `staticBikeParkAndRide`, `platformEntriesLinking`, `customNamer` and `maxAreaNodes` to the constructor as well? Then we could avoid the Supplier-thing and we could make the fields private. IMO it's better to have an obscenely big constructor than exposing public mutable fields.

Reply:
Yeah, it's true that the public fields are modified externally but from a quick glance it looked like all the modifications were made in tests that could trivially be changed to supply the values via constructor instead.

You're right that these classes would benefit from some refactoring :) But I guess that's out of scope for this PR.